scripts/db_migrator.py

```diff
@@ -1277,7 +1276,14 @@ def main():
         socket_path = args.socket
         namespace = args.namespace
 
-        load_db_config()
+        # Can't load global config base on the resoult of is_multi_asic(), because on multi-asic device, when db_migrate.py
```

Comment:
Code change here verified by this sonic-buildimage PR: https://github.com/sonic-net/sonic-buildimage/pull/18502, all test case passed.